packages/lib/src/utils/Contracts.ts

```diff
@@ -0,0 +1,73 @@
+import glob from 'glob';
+import path from 'path';
+import ZWeb3 from '../artifacts/ZWeb3';
+import ContractFactory from '../artifacts/ContractFactory';
+
+// TODO: rename to Artifacts and move to /artifacts
+
+export default class Contracts {
+
+  private static DEFAULT_SYNC_TIMEOUT:number = 240000;
+  private static DEFAULT_BUILD_DIR:string = `${process.cwd()}/build/contracts`;
+
+  private static timeout:number = Contracts.DEFAULT_SYNC_TIMEOUT;
+  private static buildDir:string = Contracts.DEFAULT_BUILD_DIR;
+  private static artifactDefaults:any = {};
```

Comment:
these variables are still initializable by constructor right?

Reply:
I don't think so... There is no constructor implemented in this class atm, and the variables are meant to be used statically, as in:
```
Contracts.timeout = 20;
Contracts.buildDir = 'some/path';
Contracts.artifactDefaults = {...};
```